mldsa/rounding.h

```diff
@@ -23,7 +23,8 @@
  *              - int32_t *a0: pointer to output element a0
  *              - int32_t *a1: pointer to output element a1
  *
- * Reference: a1 is passed as a return value instead
+ * In the reference implementation, a1 is passed as a
```

Comment:
Keep the `Reference: ...` prefix for easy grepping

Reply:
Fixed